Developers/Writing_Bears.rst

```diff
@@ -276,3 +276,98 @@ parameter in ``HiddenResult`` should be the instance of the Bear that yields
 this result (in this case ``self``), and second argument should be the content
 we want to transfer between the Bears. Here we use a list of strings as content
 but it can be any object.
+
+More Configuration Options
+--------------------------
+coala provides settings to further configure your bear according to your needs.
+Here are all the settings you can use:
+
+``LANGUAGES``: To indicate which languages your bear supports,
```

Comment:
maybe make it a list :D ? with all the attributes.

how i'd do it: first make a list with all of them, no descriptions, and then after the list you explain each of them. ideally, if you click and of them it would take you the that attribute itself.

to make them subtitles use ~~~~~~

so:

LANGUAGES
~~~~~~~~